VisitorDetails.php

```diff
@@ -26,7 +26,7 @@ public function extendVisitorDetails(&$visitor)
         );
 
         foreach ($fields as $name => $field) {
-            $visitor[$name] = $this->details[$field];
+            $visitor[$name] = $this->details[$field] ?? '';
```

Comment:
@revoltek-daniel unfortunately we still need to support PHP 5.X (we're starting work on Matomo 4 where we will increase the min supported PHP version)... any chance you could do an `isset` or `empty` check?